### DISCUSSION TITLE
Updating npmjs landing page when github README updated?

### DISCUSSION THREAD

**Discussion by slimandslam** [community/community#145498] [npm] [answered]
I've got my npmjs package and github repo linked. Everything works great and I've done my v1.0.0 release.
However, I tweaked the README.md file in the github repo. I'd like to update the landing page on npmjs without
having to change the version and re-push the entire package. Is that possible?

**Comment by toygma** [comment]
If the connection is made correctly, it should be changed within 1-2 hours.

**Comment by monishcm** [comment, marked as answer]
There is no connection between the GitHub readme and npmjs.com readme. The only way to update the readme is to publish the package again with the updated readme file and a new version number.
